JWBestPracticeApps/AdvancedPlayer/AdvancedPlayer/AppDelegate.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AppDelegate.swift
+//  AdvancedPlayer
+//
+//  Created by Michael Salvador on 1/20/22.
+//
+
+import UIKit
+import JWPlayerKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+
+        // Add your JW Player license key.
+        // JWPlayerKitLicense.setLicenseKey( )
```

Comment:
This line should be copied from the other projects. They have the insert-code widget, which makes it obvious this method takes an argument.